Yona/Yona/Others/AddNewDevice/AddDeviceViewController.swift

```diff
@@ -128,7 +128,7 @@ class AddDeviceViewController: BaseViewController, UIScrollViewDelegate {
             let trimmedWhiteSpaceString = number.removeWhitespace()
             let trimmedString = trimmedWhiteSpaceString.removeBrackets()
             
-            if trimmedString.validateMobileNumber() == false {
+            if !trimmedString.isValidMobileNumber(){
```

Comment:
Apply the same change as requested for ``AddFriendsViewController.swift``.

Reply:
Done